Sentry.xcodeproj/project.pbxproj

```diff
@@ -5208,7 +5208,6 @@
 				MTL_ENABLE_DEBUG_INFO = INCLUDE_SOURCE;
 				MTL_FAST_MATH = YES;
 				PRODUCT_BUNDLE_IDENTIFIER = io.sentry.SentrySwiftUI;
-				SDKROOT = iphoneos;
```

Comment:
I've removed this change to allow SentrySwiftUI on macOS, but if it was intentional, please let me know.